ices/100187.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+rustc "-Zcrate-attr=feature(with_negative_coherence)" - <<'EOF'
+
+#![feature(negative_impls)]
+
+// FIXME: this should compile
+
+trait MyPredicate<'a> {}
+impl<'a, T> !MyPredicate<'a> for &T where T: 'a {}
+trait MyTrait<'a> {}
+impl<'a, T: MyPredicate<'a>> MyTrait<'a> for T {}
+impl<'a, T> MyTrait<'a> for &'a T {}
+//~^ ERROR: conflicting implementations of trait `MyTrait<'_>` for type `&_`
+
+fn main() {}
+
+EOF
+
```

Comment:
This one looks like 99662

Reply:
oops, I usually copy a preexisting .sh and fill in the new ICE, looks like I forgot the latter :upside_down_face: